lib/src/sdk/dart.dart

```diff
@@ -50,5 +75,17 @@ class DartSdk extends Sdk {
   String get rootDirectory => _rootDirectory;
 
   @override
-  String? packagePath(String name) => null;
+  String? packagePath(String name) {
+    if (!isAvailable) return null;
+    var sdkPackages = _sdkPackages;
+    if (sdkPackages == null) return null;
+
+    var package =
+        sdkPackages.packages.firstWhereOrNull((pkg) => pkg.name == name);
+    if (package == null) return null;
+    var packagePath = p.joinAll([_rootDirectory, ...p.url.split(package.path)]);
```

Comment:
Isn't this just a (very) complicated way to say `package.path.split('/')`?
The only difference betewen that and `p.url.split` is that if `package.path` starts with `/`, the first element returned by `split` is `/`.

But `package.path` should always be relative. If it isn't, that `/` is not going to help.

(Does answer my question from above about which path separator it uses.)